src/app/components/topics/topics.component.html

```diff
@@ -16,8 +16,8 @@ <h4>{{ subtitle }}</h4>
                 <tr [class.table-primary]="expanded[category.name]" (click)="toggleChildTopics(category)">
                     <th>
                             <span class="expanded-icon">
-                                <fa-icon [icon]=faCaretRight *ngIf="!expanded[category.name]"></fa-icon>
-                                <fa-icon [icon]=faCaretDown *ngIf="expanded[category.name]"></fa-icon>
+<!--                                <fa-icon [icon]=faCaretRight *ngIf="!expanded[category.name]"></fa-icon>-->
+<!--                                <fa-icon [icon]=faCaretDown *ngIf="expanded[category.name]"></fa-icon>-->
```

Comment:
whats the deal with these ones? Should we replace it with tui icons?

Reply:
What I found is that I hadn't refactored this page yet. So to avoid testing failures and problems when removing font awesome icons, I would just comment them out for the meantime.